apps/src/templates/studioHomepages/UiTip.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+const UiTip = React.createClass({
+  propTypes: {
+    index: React.PropTypes.number,
+    position: React.PropTypes.object,
+    text: React.PropTypes.string,
+    closeClicked: React.PropTypes.func.isRequired
+  },
+
+  closeClicked(index) {
+    this.props.closeClicked(index);
+  },
+
+  render() {
+    const { index, position, text } = this.props;
+
+    return (
+      <div>
+        <div
+          className="arrow_box_up"
+          onClick={this.closeClicked.bind(this, index)}
```

Comment:
when using `React.createClass`, all the methods are automatically bound to `this` for you. So you shouldn't need to bind it here.